tools/ARIAtools/ARIAProduct.py

```diff
@@ -65,6 +65,17 @@ def __init__(self, filearg, bbox=None, workdir='./', verbose=False):
                 self.files=self.glob.glob(os.path.expanduser(os.path.expandvars(filearg)))
             # Convert relative paths to absolute paths
             self.files=[os.path.abspath(i) for i in self.files]
+
+        # If URLs, append with '/vsicurl/'
+        self.files=['/vsicurl/{}'.format(i) if 'https://' in i else i for i in self.files]
+        # must configure gdal to load URLs
+        if any("https://" in i for i in self.files):
+            gdal.SetConfigOption('GDAL_HTTP_COOKIEFILE','cookies.txt')
+            #check if reader is being captured as netcdf
+            [s for s in self.files if 'https://' in s][0]
```

Comment:
L75 might  have been used for debugging?

Reply:
Yup, will have to remove. Will do now.